antlr-kotlin-runtime-common/src/main/kotlin/com/strumenta/kotlinmultiplatform/Misc.kt

```diff
@@ -27,12 +27,24 @@ expect object Arrays {
 
     fun equals(a: Array<*>, b: Array<*>): Boolean
 
+    fun equals(a: IntArray, b: IntArray): Boolean
+
     fun toString(a: Array<*>): String
 }
 
-expect class BitSet {
-    constructor()
+fun Arrays.equals(a: Array<*>?, b: Array<*>?): Boolean {
+    if (a == null && b == null) return true
+    if (a == null && b != null || a != null && b == null) return false
+    return equals(a!!, b!!)
+}
 
+fun Arrays.equals(a: IntArray?, b: IntArray?): Boolean {
+    if (a == null && b == null) return true
+    if (a == null && b != null || a != null && b == null) return false
+    return equals(a!!, b!!)
+}
+
+expect class BitSet() {
```

Comment:
why did you remove the constructor from BitSet?

Reply:
I only moved the constructor so it is right after the classname 'BitSet()'. There is no difference semantically.